bookkeeper-server/src/test/java/org/apache/bookkeeper/client/BookieWriteLedgerTest.java

```diff
@@ -172,6 +177,75 @@ public void testLedgerCreateAdv() throws Exception {
         lh.close();
     }
 
+    private static boolean areByteArrayValMapsEqual(Map<String, byte[]> first, Map<String, byte[]> second) {
```

Comment:
Could you make this a general util function to make it available for both LedgerMetadata and tests?